### PR TITLE
Last Fixes

### PR DESCRIPTION
**Pull request by AdrienRem**
DemoSpeedup is to get a smoother history path

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/CoronaGame.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     public static boolean IS_ONLINE = TRUE;
     public static boolean IS_NETWORK_DEBUG = FALSE;
     private static Context context;
-    private static int DEMO_SPEEDUP = 1;
+    private static int DEMO_SPEEDUP = 10;
```
Comment (TheCrazyKing) [inline app/src/main/java/ch/epfl/sdp/CoronaGame.java]: Why put the demo speedup on the master ?
I think it suffices that if someone wants to make a demo he changes it locally.

**Review by TheCrazyKing — CHANGES_REQUESTED** [review]
What do you mean ?

File: app/src/main/java/ch/epfl/sdp/CoronaGame.java (hunk 1 above)
Comment (AdrienRem) [reply app/src/main/java/ch/epfl/sdp/CoronaGame.java]: I'd like speedup the released APK and that's a way to do it without changing all the constants

**Review by lucieho — APPROVED** [review]

**Review by TheCrazyKing — APPROVED** [review]
Nice !

**Pull request merged**